src/backend/web/routes/feed.js

```diff
@@ -67,8 +67,8 @@ const feedRoute = type => async (req, res) => {
       content: post.html,
       author: [
         {
-          name: post.author,
-          link: post.site,
+          name: post.feed.author,
+          // TODO: add site info for this feed https://github.com/Seneca-CDOT/telescope/issues/724
```

Comment:
this will be for gatsby correct?

Reply:
No, this part, the `/feed/rss` type routes are for generating an RSS/ATOM/etc feed for our post content.  That is, if someone wanted to import all the blogs in our system into their feed reader, they would use this.